From 6c7a6485de4b5b36df8e34a5582ebcb0e55dfd2d Mon Sep 17 00:00:00 2001
From: "Rich.jiangyk" <jyk@rock-chips.com>
Date: Mon, 13 Sep 2010 17:16:27 +0800
Subject: [PATCH] fix ov9650 init_array to solve mirrored images

ov9650 register 0x1e controls mirro/vflip functions. modify settings to correct mirrored image problems.
---
 drivers/media/video/ov9650.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100644 => 100755 drivers/media/video/ov9650.c

diff --git a/drivers/media/video/ov9650.c b/drivers/media/video/ov9650.c
old mode 100644
new mode 100755
index 4930818e1c77..f4e9aac274b1
--- a/drivers/media/video/ov9650.c
+++ b/drivers/media/video/ov9650.c
@@ -113,7 +113,7 @@ static struct reginfo sensor_init_data[] =
     {0x37, 0x00},
     {0x35, 0x81},
     {0x0e, 0x20},
-    {0x1e, 0x04},//34：第一个水平线有异常 -> ;//04: <- ;//14: ;//旋转度
+    {0x1e, 0x24},//34：第一个水平线有异常 -> ;//04: <- ;//14: ;//旋转度
     {0xA8, 0x80},
     {0x12, 0x10},
     {0x04, 0x00},
-- 
2.35.3

